tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -450,3 +450,39 @@ sudo umount $MOUNT_DIR
 mount.gcsfuse $TEST_BUCKET_NAME $MOUNT_DIR -o implicit_dirs -o config_file=/tmp/gcsfuse_config.yaml
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/managed_folders/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR --testbucket=$TEST_BUCKET_NAME -run TestEnableEmptyManagedFoldersTrue
 sudo umount $MOUNT_DIR
+
+# For GRPC: running only core integration tests.
+
+# Test packages: operations
+# Run test with static mounting. (flags: --client-protocol=grpc --implicit-dirs=true)
+gcsfuse --client-protocol=grpc --implicit-dirs=true $TEST_BUCKET_NAME $MOUNT_DIR
```

Comment:
Regarding sharing the gRPC-related GCSFuse package with the GKE team:
Is the GKE team building the package from the master branch? As grpc related changes are not roll out yet, so on which gcsfuse package they will run tests?

PTAL

Reply:
Ohh, you mean adding here can break the GCSFuse integration pipeline?

What's your recommendation?

Reply:
Discussed offline to check with GKE team.